main.py

```diff
@@ -382,6 +389,12 @@ def main():
         "--read-only",
         action='store_true',
         help="Es wird nicht nach fehlenden Kontaktdaten gefragt. Stattdessen wird ein Fehler angezeigt, falls benötigte Kontaktdaten in der JSON-Datei fehlen.")
+    base_subparser.add_argument(
+        "--t-token",
+        help="Token des Telegram-Bot")
+    base_subparser.add_argument(
+        "--t-id",
+        help="ChatID an die die benachrichtigungen gesendet werden sollen")
```

Comment:
Lange Optionen dürfen lang sein. Und bitte Grammatik beachten.
```suggestion
    base_subparser.add_argument(
        "--telegram-token",
        help="Token des Telegram-Bots")
    base_subparser.add_argument(
        "--telegram-id",
        help="ChatID, an die Benachrichtigungen gesendet werden sollen")
```

Reply:
Wenn es es nur die Datei Option gibt braucht man den code hier nicht mehr

Reply:
> > @TheMrPhantom besteht die Möglichkeit das du nicht auf das Repo verweist sondern direkt die notwendigen Files unter tools/PythonTelegramWraper ablegst?
> 
> @sinun98 Klar besteht die Möglichkeit. Das ist aber eine sau schlechte Idee, da so nicht direkt klar ist, wo das herkommt und welche Version verwendet wird. Außerdem sind Updates dann schwierig und führen hier zu einem Diff der größer ist als nötig.
> 
> Stattdessen sollte das Submodul einfach per HTTPS eingebunden werden, damit es in jeder Umgebung mit Internetzugang initialisiert werden kann.

Das mit HTTPS hört sich gut an